packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.7.6
+* Implement UIAlertController with a preferredStyle of UIAlertControllerStyleAlert since UIAlertView is deprecated.
```

Comment:
nit: add back-ticks to indicate the `UIAlertController`, `UIAlertControllerStyleAlert` and `UIAlertView` are code constructs:

```suggestion
* Implement `UIAlertController` with a preferredStyle of `UIAlertControllerStyleAlert` since `UIAlertView` is deprecated.
```

Reply:
done.